src/features/components/templates/navigation/template/desktop-menu/desktop.items.tsx

```diff
@@ -4,26 +4,34 @@ import { SmartNavDropItems } from '../../types'
 import NavDropItems from '../nav-drop-items'
 import { Localize } from 'components/localization'
 import Typography from 'features/components/atoms/typography'
+import dclsx from 'features/utils/dclsx'
 
 interface DesktopItemsProps {
     item: SmartNavDropItems
+    active: string
 }
 
-const NavDesktopItem = ({ item }: DesktopItemsProps) => {
+const NavDesktopItem = ({ item, active }: DesktopItemsProps) => {
     const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
         if (!window.matchMedia('(hover: none)').matches) {
             e.preventDefault()
         }
     }
 
+    const has_scroll_menu = active === '_t_Trade_t_' || active === '_t_Markets_t_'
```

Comment:
Hi @mohsen-deriv just one concern. Maybe you can ignore. 
What if management increase `About us` OR `Resources` nav as well.  😬

Reply:
ohh if that's the case then you are right. 
Thank @mohsen-deriv 🙏